.babelrc

```diff
@@ -1,12 +1,31 @@
 {
-  presets: ['@babel/react', '@babel/env', ["@babel/preset-stage-2", { "decoratorsLegacy": true }]]
-  /*
+  "presets": [
+    "@babel/react",
+    "@babel/env"
+    /*
     Babel uses these "presets" to know how to transpile your Javascript code. Here's what we're saying with these:
-
```

Comment:
is there a reason for losing this line break? It seems arbitrary.

Reply:
No, just a copy/paste issue when putting the .babelrc back together.